packages/trpc/server/routers/viewer/bookings.tsx

```diff
@@ -194,7 +238,7 @@ export const bookingsRouter = router({
               },
             },
           ],
-          AND: [passedBookingsFilter],
+          AND: [passedBookingsStatusFilter, ...filtersCombined],
```

Comment:
You're potentially spreading undefined here, right? Not sure if that would work or you need to default to an empty array on line 206.